pom.xml

```diff
@@ -37,6 +37,21 @@
             <version>7.15.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.4.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>2.0.5</version>
+        </dependency>
```

Comment:
Why are we including both the logback-core and slf4j-api dependencies? I believe using just the logback-classic dependency would be sufficient for our logging needs.

Reply:
was looking into this article while adding dependencies https://www.baeldung.com/logback